backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/objcexport/ObjCExportCodeGenerator.kt

```diff
@@ -962,7 +960,7 @@ private fun ObjCExportCodeGenerator.createTypeAdapter(
 
     when (descriptor.kind) {
         ClassKind.OBJECT -> {
-            classAdapters += if (descriptor.isUnit()) {
+            classAdapters += if (descriptor.isUnit() || descriptor.isNothing()) {
```

Comment:
This code handles `descriptor.kind == ClassKind.OBJECT` case.
Is there `object Nothing`?